README.md

```diff
@@ -1,111 +1,108 @@
 # nm-configurator
 
-A tool capable of identifying & storing the relevant NetworkManager settings
-for a given host out of a pool of predefined desired configurations.
+nm-configurator (or NMC) is a CLI which makes it easy to generate and apply NetworkManager configurations.
 
-Typically used with [Combustion](https://documentation.suse.com/sle-micro/5.4/single-html/SLE-Micro-deployment/#cha-images-combustion) 
-in order to bootstrap multiple nodes using the same provisioning artefact instead of depending on different custom images per machine.
-
-## What are the prerequisites?
+## How to install it?
 
-### Desired network configurations per host
+### Standard method
 
-`nm-configurator` depends on having the desired network state for all known nodes beforehand.
+Each release is published with NMC already built for `amd64` and `arm64` Linux systems:
 
-[NetworkManager](https://documentation.suse.com/sle-micro/5.4/html/SLE-Micro-all/cha-nm-configuration.html) 
-is using connection profiles defined as files stored under `/etc/NetworkManager/system-connections`.
-These config files (*.nmconnection) can be generated using [nmstate](https://nmstate.io/features/gen_conf.html).
+For AMD64 / x86_64 based systems:
 
-Each file contains the desired state for a single network interface (e.g. `eth0`).
-Configurations for all interfaces for all known hosts must be generated using `nmstate`.
+```shell
+$ curl -o nmc -L https://github.com/suse-edge/nm-configurator/releases/latest/download/nmc-x86_64 
+$ chmod +x nmc
+```
 
-### Network interface mapping
+For ARM64 / aarch64 based systems:
 
-Network interface mapping is required in order for `nm-configurator`
-to identify the proper configurations for each host it is running on.
+```shell
+$ curl -o nmc -L https://github.com/suse-edge/nm-configurator/releases/latest/download/nmc-aarch64
+$ chmod +x nmc
+```
 
-This additional config must be provided in a YAML format mapping the logical name of the interface to its MAC address:
+### Manual method
 
-```yaml
-host_config:
-  - hostname: node1.example.com
-    interfaces:
-      - logical_name: eth0
-        mac_address: 00:10:20:30:40:50
-      - logical_name: eth1
-        mac_address: 10:20:30:40:50:60        
-  - hostname: node2.example.com
-    interfaces:
-      - logical_name: eth0
-        mac_address: 00:11:22:33:44:55
+```shell
+$ git clone https://github.com/suse-edge/nm-configurator.git
+$ cd nm-configurator
+$ cargo build --release # optionally specify --target flag if cross compiling
```

Comment:
It will not compile on macOS. There are differences between libc on Mac and on Linux but I didn't dive deeper into it since I didn't think tests on macOS would be intended. Should we look further into this?

Reply:
Or maybe just put a disclaimer that this tool is intended to be executed on linux only.